plugin/pkg/admission/security/podsecurity/testdata/pod.yaml

```diff
@@ -0,0 +1,675 @@
+apiVersion: v1
```

Comment:
It looks like this pod passes the restricted check? Consider adding a header comment or renaming to `restricted-pod.yaml`. I assume we'd want to update it if we ever add a restriction to restricted that causes this to fail the check?

Reply:
added a comment at the top